packages/core/src/constants/chains.ts

```diff
@@ -290,6 +291,29 @@ export const arbitrumRinkeby: Chain = {
   testnet: true,
 }
 
+export const arbitrumGoerli: Chain = {
+  id: chainId.arbitrumGoerli,
+  name: 'Arbitrum Goerli',
+  network: 'arbitrum-goerli',
+  nativeCurrency: {
+    name: 'Arbitrum Goerli Ether',
+    symbol: 'AGETH',
+    decimals: 18,
+  },
+  rpcUrls: {
+    default: publicRpcUrls.arbitrumGoerli,
+    public: publicRpcUrls.arbitrumGoerli,
+  },
+  blockExplorers: {
+    default: {
+      name: 'Arbitrum Explorer',
+      url: 'https://goerli-rollup-explorer.arbitrum.io',
+    },
+  },
+  multicall: undefined,
```

Comment:
Let's update this once [Multicall3 is deployed](https://github.com/mds1/multicall/issues/38).